### ISSUE TITLE
Drag And Drop does not work with event firing

### ISSUE THREAD

**Issue by sebastianciupinski** [robotframework/SeleniumLibrary#1653]
# Steps to reproduce the issue

```*** Setting ***
Library    SeleniumLibrary

*** Variables ***
${SELENIUM_SPEED}		1

*** Test Cases ***

A
    Open Browser   http://the-internet.herokuapp.com/  gc     options=add_argument("--ignore-certificate-errors")
    Set Selenium Speed     ${SELENIUM_SPEED}
    Capture Page Screenshot
    Click Link     Dropdown
    Go Back
    Click Link     Drag and Drop
    ${a}=       Get Webelement     column-a
    ${b}=       Get Webelement     column-b
    Drag And Drop    ${a}    ${b}
    Drag And Drop    ${b}    ${a}
    Close Browser
```

Such code works, slowly and requires cursor to be in browser window, but works
${a} and ${b} are `selenium.webdriver.remote.webelement.WebElement`

While having the same code with

```
Library    SeleniumLibrary    event_firing_webdriver=${CURDIR}${/}SomeScript.py
```

code does not work, because Drag And Drop requires webelement object, while having event firing driver, ${a} and ${b} are `selenium.webdriver.support.event_firing_webdriver.EventFiringWebElement` and result is: `AttributeError: move_to requires a WebElement`

```
robotframework==3.2.2
robotframework-seleniumlibrary==4.5.0
selenium==3.141.0
chrome 85.0.4183.121 on windows 10
```

**Comment by aaltat** [comment]
This is actually bug in Selenium side. But we have been making workarounds for similar type of problems also in other places in SeleniumLibrary code  so we could fix it. 

As a quick fix, you could try out Selenium 4 pre releases. In there this problem should be fixed. Please remember that Selenium is still in Alpha phase, so it might not work correctly always, but is worth of try.

**Comment by aaltat** [comment]
Fixed in #1679
